docs/versionutils.py

```diff
@@ -0,0 +1,177 @@
+        proc = subprocess.Popen(cmd, stdout=subprocess.PIPE,
+                                stderr=subprocess.PIPE,
+                                cwd=self.repo_root)
+        stdout, stderr = proc.communicate()
+        if proc.returncode > 0:
+            logger.warn("%s failed with:\nstdout:\n%s\nstderr:\n%s\n"
+                        % (cmd, stdout, stderr))
+            return ''
+        return stdout.decode('utf8')
+
+    def get_versions(self, tags):
+        versions = {}
+        for tag in tags:
+            version = {}
+            setup_py = self._get_setup_py(tag)
+            for package in self.headers[1:] + ['qiskit_terra']:
+                version_regex = re.compile(package + '[=|>]=(.*)\"')
+                match = version_regex.search(setup_py)
+                if match:
+                    ver = match[1]
+                    if '<' in match[1]:
+                        ver = '>=' + ver
+                    if package != 'qiskit_terra':
+                        version[package] = ver
+                    else:
+                        version['qiskit-terra'] = ver
+            if version:
+                versions[tag] = version
+        return versions
+
+    def build_table(self, versions):
+        table = nodes.table()
+        table['classes'] += ['colwidths-auto']
+        tgroup = nodes.tgroup(cols=len(self.headers))
+        table += tgroup
+        self.options['widths'] = [30, 15, 15, 15, 20, 15]
+        tgroup.extend(
+            nodes.colspec(colwidth=col_width, colname='c' + str(idx))
+            for idx, col_width in enumerate(self.col_widths)
+        )
+
+        thead = nodes.thead()
+        tgroup += thead
+
+        row_node = nodes.row()
+        thead += row_node
+        row_node.extend(nodes.entry(h, nodes.paragraph(text=h))
+                        for h in self.headers)
+
+        tbody = nodes.tbody()
+        tgroup += tbody
+
+        rows = []
+        for version in versions:
+            row_node = nodes.row()
+            entry = nodes.entry()
+            entry += nodes.paragraph(text=version)
+            row_node += entry
+            for cell in self.headers[1:]:
+                if cell in versions[version]:
+                    entry = nodes.entry()
+                    text = versions[version][cell]
+                    entry += nodes.paragraph(text=text)
+                else:
+                    entry = nodes.entry()
+                row_node += entry
+            rows.append(row_node)
+        tbody.extend(rows)
+        return table
+
+    def run(self):
+        tags = _get_git_tags()
+        versions = self.get_versions(tags)
+        self.max_cols = len(self.headers)
+        self.col_widths = self.get_column_widths(self.max_cols)
+        table_node = self.build_table(versions)
+        title, messages = self.make_title()
+        if title:
+            table_node.insert(0, title)
+        return [table_node] + messages
```

Comment:
```suggestion
        return [table_node] + messages
        
```
Adding new line to the end of file